network/server_test.go

```diff
@@ -41,14 +41,14 @@ func TestConnLimit_Inbound(t *testing.T) {
 		t.Fatalf("Unable to join servers, %v", joinErr)
 	}
 
-	// Server 2 tries to connect to Server 0
-	// but Server 0 is already connected to max peers
+	// Server 2 tries to connect to Server 1
+	// but Server 1 is already connected to max inbound peers
 	smallTimeout := time.Second * 5
-	if joinErr := JoinAndWait(servers[2], servers[0], smallTimeout, smallTimeout); joinErr == nil {
-		t.Fatal("Peer join should've failed")
+	if joinErr := JoinAndWait(servers[2], servers[1], smallTimeout, smallTimeout); joinErr == nil {
```

Comment:
Why did you change the servers being joined here? Server 0 should also be at max inbound peers after the previous join call in the test

Reply:
Resolving this because Server 0 will have made an _outbound_ call to server 1, not _inbound_